src/Tribe/Privacy.php

```diff
@@ -54,4 +56,176 @@ public function default_privacy_policy_content() {
 		return apply_filters( 'tribe_tickets_default_privacy_policy_content', $content );
 
 	}
+
+	/**
+	 * Register exporter for Tickets attendees saved data.
+	 *
+	 * @since TBD
+	 * @param $exporters
+	 *
+	 * @return array
+	 */
+	public function register_exporter( $exporters ) {
+		$exporters[] = array(
+			'exporter_friendly_name' => __( 'Event Ticket RSVP Atendee' ),
+			'callback'               => array( $this, 'rsvp_exporter' ),
+		);
+
+		$exporters[] = array(
+			'exporter_friendly_name' => __( 'Event Ticket TribeCommerce Atendee' ),
+			'callback'               => array( $this, 'tpp_exporter' ),
+		);
+
+		return $exporters;
+	}
+
+	/**
+	 * Exporter for Events Ticket RSVP Attendee
+	 *
+	 * @param     $email_address
+	 * @param int $page
+	 * @since     TBD
+	 *
+	 * @return array
+	 */
+	public function rsvp_exporter( $email_address, $page = 1 ) {
+		$number = 500; // Limit us to avoid timing out
```

Comment:
Do we need to add a way to make export more later? if so lets add a `@todo` in code too.

Reply:
No, the idea is that is done in batches, and that's the number of each batch. WordPress added this, so the request for retrieving personal data doesn't timeout (plugins and themes can hook in there.)